doc/plugin_server_upstreamauthority_vault.md

```diff
@@ -23,6 +23,24 @@ The plugin supports **Client Certificate**, **Token** and **AppRole** authentica
 - **Token** method authenticates to Vault using the token in a HTTP Request header.
 - **AppRole** method authenticates to Vault using a RoleID and SecretID that are issued from Vault.
 
+The PKI secret engine must configure its TTL higher than `ca_ttl`(server configuration).
```

Comment:
I think that we should say "higher than or equal to". Adding a link to spire_server.md may be appropriate also.
I'm wondering if it's worth noting that this can be adjusted in either side?
i.e.: the `ca_ttl` SPIRE Server configurable should be less than or equal to the Vault's PKI secret engine TTL.

Reply:
>  I'm wondering if it's worth noting that this can be adjusted in either side?

The Vault has two TTLs: a global TTL and a TTL that can be configured on a Role.
If the TTL is not set on the Role, the global TTL will be used.

i.e.
> Increase the TTL by tuning the secrets engine. The default value of 30 days may be too short, so increase it to 1 year:

> $ vault secrets tune -max-lease-ttl=8760h pki
Success! Tuned the secrets engine at: pki/

> Note that individual roles can restrict this value to be shorter on a per-certificate basis. This just configures the global maximum for this secrets engine.